Python/fileutils.c

```diff
@@ -1320,6 +1324,18 @@ _Py_get_inheritable(int fd)
 static int
 set_inheritable(int fd, int inheritable, int raise, int *atomic_flag_works)
 {
+#ifdef MS_WINDOWS_NON_DESKTOP
```

Comment:
Note for reference if turns out we can support `SetHandleInformation()` if `HANDLE_FLAG_INHERIT` is manually defined.

Reply:
added back by manually defining `HANDLE_FLAG_INHERIT`